AlphaWallet/Transfer/ViewModels/TransactionConfirmation/SwapTransactionViewModel.swift

```diff
@@ -72,13 +72,15 @@ extension TransactionConfirmationViewModel {
                     return .init(title: .normal(configurationTitle), headerName: headerName, details: gasFee, configuration: configuration)
                 }
             case .from:
-                let amount = EtherNumberFormatter.short.string(from: BigInt(fromAmount), decimals: fromToken.decimals)
-                let symbol = fromToken.symbol
-                return .init(title: .normal("\(amount) \(symbol)"), headerName: headerName, configuration: configuration)
+                let doubleAmount = (Decimal(bigInt: BigInt(fromAmount), decimals: fromToken.decimals) ?? .zero).doubleValue
+                let amount = NumberFormatter.shortCrypto.string(double: doubleAmount, minimumFractionDigits: 4, maximumFractionDigits: 8)
```

Comment:
I think we might have the `4` as a constant somewhere. If we haven't, we should (8 too).